src/config.ts

```diff
@@ -175,3 +175,8 @@ export const ORGANIZATION_ID_DEFAULT = getEnv('ORGANIZATION_ID_DEFAULT', null);
 export const ORGANIZATION_OWNER_ID_DEFAULT = getEnv('ORGANIZATION_OWNER_ID_DEFAULT');
 export const PROJECT_ID_DEFAULT = getEnv('PROJECT_ID_DEFAULT', null);
 export const PROJECT_ADMIN_ID_DEFAULT = getEnv('PROJECT_ADMIN_ID_DEFAULT');
+
+export const RUNS_QUOTA_DAILY = parseInt(getEnv('RUNS_QUOTA_DAILY', '100'));
```

Comment:
Please add to the .env.example

Reply:
I was thinking about keeping it intentionally hidden in order to not further extend the set of environment variables. What do you think?

Reply:
It can be only internal but then we should set the limit to the infinite.